.github/workflows/build.yml

```diff
@@ -19,6 +21,8 @@ jobs:
           GH_BUILD_PASSWORD: ${{ secrets.GH_BUILD_PASSWORD }}
       run: |
         TAG=$(date +%Y-%m-%d)-$(git rev-parse --short HEAD)
+        [[ "${{ github.ref }}" == "refs/tags/"* ]] && TAG=$(echo "${{ github.ref }}" | sed 's/refs\/tags\/[v]//')
```

Comment:
Just want to confirm lines similar to line 24 will only get run on tagged commits and that untagged commits will use the TAG from line 23.

Reply:
Yup, that's the intention. :+1: